api/catalog/api/views/image_views.py

```diff
@@ -107,17 +106,7 @@ def oembed(self, request, *_, **__):
     )
     def thumbnail(self, request, *_, **__):
         image = self.get_object()
-
-        image_url = image.url
-        if not image_url:
-            raise NotFound("Could not find image.", 404)
```

Comment:
Is removing this check intentional/are we just relying on the upstream response from photon to return 404 in this case?